packages/react-storage/src/components/StorageBrowser/composables/ActionCancel.tsx

```diff
@@ -0,0 +1,30 @@
+import React from 'react';
+
+import { ButtonElement, IconElement } from '../context/elements';
+import { CLASS_BASE } from '../views/constants';
+
+export interface ActionCancelProps {
+  onCancel?: () => void;
+  isDisabled?: boolean;
+  label?: string;
+}
+
+export const ActionCancel = ({
+  onCancel,
+  isDisabled,
+  label,
+}: ActionCancelProps): React.JSX.Element => (
+  <ButtonElement
+    variant="cancel"
+    className={`${CLASS_BASE}__action-cancel`}
+    onClick={onCancel}
+    disabled={isDisabled}
+  >
+    {label ?? (
+      <IconElement
```

Comment:
I don't think we're using the IconElement are we?

Reply:
True, since I removed the table's cancel button, don't need this Icon here.